types/artifacts.d.ts

```diff
@@ -158,6 +158,7 @@ declare global {
       export type MetaElement = LH.Artifacts['MetaElements'][0];
 
       export interface NodeDetails {
+        id: string;
```

Comment:
ok so `id` conflicts with some types that extend NodeDetails (`ScriptElement`). what else should this be named? `lhId`? And if things like ScriptElement or LinkElement will extend this interface, perhaps this field should also be optional.